misk-aws-dynamodb-testing/src/main/kotlin/misk/aws/dynamodb/testing/CreateTablesService.kt

```diff
@@ -47,8 +46,16 @@ internal class CreateTablesService @Inject constructor(
       // Provisioned throughput needs to be specified when creating the table.
       globalSecondaryIndex.provisionedThroughput = ProvisionedThroughput(1L, 1L)
     }
+    tableRequest = table.customizeRequest(tableRequest)
+
     DynamoDB(this).createTable(tableRequest).waitForActive()
   }
+
+  companion object {
+    val customizeRequestNoop: (CreateTableRequest) -> CreateTableRequest = {
```

Comment:
object val properties with no custom get function that hold deeply immutable data) should use uppercase underscore-separated names:

 https://kotlinlang.org/docs/reference/coding-conventions.html#property-names